samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -266,7 +258,8 @@ Testing with nRF Connect for Desktop
 
 1. Connect the development kit to the computer using a USB cable.
    The computer assigns to the development kit a COM port on Windows or a ttyACM device on Linux, which is visible in the Device Manager.
-#. |connect_terminal|
+#. Connect to the kit with a terminal emulator.
+   See :ref:`Direct Test Mode terminal connection` for the required settings.
```

Comment:
```suggestion
   See :ref:`Direct Test Mode terminal connection`_ for the required settings.
```

Reply:
Changed. It should be `Direct Test Mode terminal connection`_ without ":ref:"

Reply:
You are absolutely right.